README.md

```diff
@@ -4,6 +4,25 @@ git clone https://github.com/theforeman/smart-proxy
 ```
 Configure smart proxy
 
+
+In smart-proxy directory 
+```shell
+mkdir smart-proxy/logs
```

Comment:
This should be just `mkdir logs` since you're running it from the smart proxy directory